source/assets/components/shoppingCard/shoppingCard.js

```diff
@@ -33,7 +33,7 @@ class shoppingCard extends HTMLElement {
         <div class="card-header" id="recipe-title">${recipeTitle}</div>
         <div class="card-body" id="recipe-ingredients"></div>
         <div class="card-footer border-0 pt-0 mt-0">
-          <img id = "fav-btn" src="assets/icons/shopping/trash-can.png" width = "50px" height = "50px" alt="trash-can-icon">
+          <img id = "fav-btn" src="assets/icons/shopping/trash-can.png" width = "50rem" height = "50rem" alt="trash-can-icon">
```

Comment:
We should change the `img id = "fav-btn` to something else like `img id = "trash-btn`

Reply:
Done